bundles/org.openhab.binding.awattar/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,401 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="awattar"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="awattar-bridge">
+		<label>aWATTar Bridge</label>
+		<description>Provides price data from the aWATTar API.</description>
+		<config-description uri="bridge-type:awattar:awattar-bridge">
+			<parameter name="country" type="text">
```

Comment:
See https://www.openhab.org/docs/developer/bindings/config-xml.html also for min/max values.

```suggestion
			<parameter name="country" type="text" pattern="DE|AT">
```

Reply:
Changed